CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Changelog
 
+# 13.0.0
+
+#### Breaking changes
+
+- In variables:
+  - Remove SimpleVariable and DatedVariable. Any Variable acts like old DatedVariable
+  - Remove start_date attribute
+  - Rename stop_date attribute to end
+- In formulas:
+  - Remove SimpleFormula and DatedFormula. Any Formula acts like old DatedFormula
+  - Remove @dated_function: start goes to formula name suffix, stop is deduced from other formulas start dates and variable end attribute 
+
+#### New features
+
+- Introduce `formula_YYYY[_MM[_DD]]`
```

Comment:
This replaces the former `@dated_function` decorator, so it should be mentioned near the its deprecation.

As a user, it's better for me to find in the same place what syntax has disappeared, and what has replaced it.